libphal/phal_dump.C

```diff
@@ -123,6 +123,58 @@ uint32_t getFapiUnitPos(pdbg_target* target)
 		log(level::ERROR, "ATTR_FAPI_POS Attribute get failed");
 
 	return fapiUnitPos;
+
```

Comment:
Why do I see the function starting before getFapiUnitPos last closing braces.
Is this compiled and tested? Clang format applied?